lib/ProductOpener/Display.pm

```diff
@@ -342,6 +342,37 @@ sub process_template ($template_filename, $template_data_ref, $result_content_re
 	$template_data_ref->{static_subdomain} = $static_subdomain;
 	$template_data_ref->{images_subdomain} = $images_subdomain;
 	$template_data_ref->{formatted_subdomain} = $formatted_subdomain;
+
+	# Read's Release version from version.txt file & Link's the relsease notes of openfoodfacts-bot
+
+	my $version = "";
```

Comment:
Regarding separation of concerns, it doesn't seem like it should be the concern of this method to read the `version.txt` file and to build the link. The version itself is a tiny bit of info that will not change during the runtime of the Perl process, so it could probably be done in `lib/startup_apache2.pl`, potentially in https://github.com/openfoodfacts/openfoodfacts-server/blob/bea5720420347b16cb7f7b99c3adbfde098d971e/lib/ProductOpener/LoadData.pm#L75.